cloudbuild.yaml

```
@@ -0,0 +1,79 @@
+        fi
+
+  # Step 5: Deploy to Firebase Hosting
+  - name: 'gcr.io/$PROJECT_ID/firebase'
+    args:
+      - 'deploy'
+      - '--only'
+      - 'hosting:${_FIREBASE_SITE_ID}'
+      - '--project'
+      - '${_FIREBASE_PROJECT_ID}'
+      - '--token'
+      - '${_FIREBASE_TOKEN}'
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)